bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/package-info.java

```diff
@@ -34,7 +34,7 @@
  *      version, is bound to this proxy component resource type.
  * </p>
  */
-@Version("12.14.0")
+@Version("13.0.0")
```

Comment:
Why this bump?

Reply:
Because of the modifications required to the interface, due to adhering to the earlier request to not introduce a new version, a bump was required by the build tooling.